src/test/java/ibm/jceplus/junit/openjceplus/integration/TestAll.java

```diff
@@ -7,11 +7,10 @@
  */
 package ibm.jceplus.junit.openjceplus.integration;
 
-import org.junit.platform.suite.api.SelectClasses;
 import org.junit.platform.suite.api.Suite;
+import org.junit.platform.suite.api.SelectPackages;
 
```

Comment:
think the order of SelectPackages should be front of Suites alphabetically?

Reply:
Yes we do try to keep the imports in alphabetic order. I restored the order to the old one in this case.

Reply:
I also opened PR https://github.com/IBM/OpenJCEPlus/pull/176 to strictly enforce import ordering in the future.